shipyard-controller/internal/handler/projecthandler.go

```diff
@@ -346,6 +346,10 @@ func (ph *ProjectHandler) CreateProject(c *gin.Context) {
 			SetConflictErrorResponse(c, err.Error())
 			return
 		}
+		if err.Error() == common.AlreadyInitializedRepositoryMsg {
```

Comment:
here we should use `strings.Contains()`, as an exact string comparison will not work for wrapped errors

Reply:
fixed